website/content/docs/enterprise/license/faq.mdx

```diff
@@ -20,6 +20,59 @@ This check is part of the server boot-up process.
 In previous versions of HashiCorp enterprise products, one server could distribute a license to other servers via the Raft protocol.
 This will no longer work since each server must be able to find a valid license during the startup process.
 
+## Q: What are the upgrade requirements?
+
+If ACLs are enabled, customers on Consul Enterprise 1.8+ or 1.9+ must upgrade Consul agents to the latest patch available (1.8.13 or 1.9.7) prior to upgrading the Consul server instances to 1.10.0.
```

Comment:
per our talk earlier today and what I understood, *any* upgrade to 1.10.0 enterprise must start with upgrading to the latest patch, so I would replace this sentence with

"Any customers on Consul Enterprise 1.8/1.9 should first upgrade their agents to the latest patch release. Additionally, the license file must be set in an environment variable or file path, and the upgrade process varies if ACLs are not enabled in the cluster. We have provided additional instructions below"

cc @mkeeler for additional review on what I said above

Reply:
That all looks correct to me.